contributing/data-model-schema.txt

```diff
@@ -29,9 +29,32 @@ wastes your time. From the top-level :file:`bioformats` folder, while
 the most thorough is :command:`git clean -dfx`.
 
 
+Major or minor release?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A minor release of the OME model schema may suffice for changes like
```

Comment:
I'm not sure they add much in this case (English "must" is already unambiguous even without the emphasis I added); I also wonder if those definitions should be a more general intro to the docs than repeated at the top of most pages, then the pages can just put them in some special style (like RFCs' all-caps)? Could certainly add here if seems warranted but I think this might be worth a deeper think beyond this PR.